code/modules/reagents/reactions/reaction_drinks.dm

```diff
@@ -47,6 +47,7 @@
 	required_reagents = list(/datum/reagent/drink/milk = 5, /datum/reagent/nutriment/coco = 1)
 	result_amount = 5
 	mix_message = "The solution thickens into a creamy brown beverage."
+	maximum_temperature = 70 CELSIUS
```

Comment:
I think you may have wanted minimum temperature for this?

Reply:
This one is max, other is min, so they don't overlap. If you want chocolate, heat it up; if  you want chocolate milk, do it cold.

Reply:
Ah fair enough.